OSDOCS:13467: Creating new tutorial book and updating CLI tutorial (#90215)
openshift/openshift-docs: modules/getting-started-cli-creating-secret.adoc

// Module included in the following assemblies:
//
// * tutorials/dev-app-cli.adoc

:_mod-docs-content-type: PROCEDURE
[id="getting-started-cli-creating-secret_{context}"]
= Providing access to the database by creating a secret

> The `nationalparks` application needs information, such as the database name, username, and passwords, on how to access the MongoDB database. However, because this is sensitive information, it should not be stored directly in the pod.

Reviewer: ```suggestion
The `nationalparks` application needs information, such as the database name, username, and passwords, to access the MongoDB database. However, because this information is sensitive, you should not store it directly in the pod.
```

You can use a _secret_ to store sensitive information, and share that secret with workloads.

> `Secret` objects provide a mechanism to hold sensitive information such as passwords, {product-title} client configuration files, private source repository credentials, and so on. Secrets decouple sensitive content from the pods. You can mount secrets into containers using a volume plugin or the system can use secrets to perform actions on behalf of a pod.

Reviewer: ```suggestion
`Secret` objects provide a mechanism to hold sensitive information such as passwords, {product-title} client configuration files, private source repository credentials, and so on. Secrets decouple sensitive content from the pods. You can mount secrets into containers by using a volume plugin, or the system can use secrets to perform actions on behalf of a pod.
```

deployment "nationalparks" successfully rolled out
----

> // TODO: I don't think mongodb-nationalparks actually rolls out a new revision... double check. Remove if not.

Reviewer 2: > @AlexonOliveiraRH I removed this check for the rollout of mongodb-nationalparks. As far as I could tell in my testing, it didn't roll out a new revision here. Are you okay with removing this?

@bergerhoffer ACK, I'm OK with this too. The command itself it's only to check if the app pod is indeed up and running, but there are other ways to check this.

> // TODO: I don't think mongodb-nationalparks actually rolls out a new revision... double check. Remove if not.

Author: Sounds good, removing!
